### PR TITLE
Fix typos in specifying-a-routes-model.md

### PR DESCRIPTION
**Pull request by locks**
- Align file name and class name
- Use singular noun for `store.findAll`

### CODE REVIEW

**Comment by ijlee2** [comment]
Thanks, @locks! Reviewing the file now to check if there are other typos that we want to fix in this pull request.

File: guides/release/routing/specifying-a-routes-model.md [hunk 1]
```diff
@@ -205,7 +205,7 @@ Whatever shows up in the URL at the `:post_id`, the dynamic segment, will be ava
 ```javascript {data-filename=app/routes/post.js}
 import Route from '@ember/routing/route';
 
-export default class PhotoRoute extends Route {
+export default class Route extends Route {
```
Comment (MelSumner) [inline guides/release/routing/specifying-a-routes-model.md:208]: If this is correct then the generator is also not working correctly.

[the remark this answers is not in the document]
Comment (locks) [reply guides/release/routing/specifying-a-routes-model.md:208]: That's what I meant to do, dunno how I lost the change!

**Review by MelSumner — APPROVED** [review]
Thank you for making these updates!

**Pull request merged**